recipe/meta.yaml

```
@@ -36,8 +36,8 @@ requirements:
     - fcitx-qt5 >=1.2.7  # [linux64]
     - fzf >=0.42.0
     - intervaltree >=3.0.2
-    - ipython >=7.31.1,<9.0.0,!=8.8.0,!=8.9.0,!=8.10.0,!=8.11.0,!=8.12.0,!=8.12.1
-    - jedi >=0.17.2,<0.19.0
+    - ipython >=8.13.0,<9.0.0,!=8.17.1
```

```suggestion
    - ipython >=8.12.2,<9.0.0,!=8.17.1
```

The 8.12 series is the last one in IPython to support Python 3.8